src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.PowerShell.Commands
     /// The ConvertTo-Json command.
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-windows`
Json Tests.ConvertTo-Json Bug Fixes.ConvertTo-JSON should not have hard coded english error message
```powershell
Expected an exception, with FullyQualifiedErrorId 'System.NotSupportedException,Microsoft.PowerShell.Commands.ConvertToJsonCommand' to be thrown, but the FullyQualifiedErrorId was 'System.Text.Json.JsonException,Microsoft.PowerShell.Commands.ConvertToJsonCommand2'. from D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Json.Tests.ps1:53 char:15
    +             { ConvertTo-Json -InputObject $hash } | Should -Throw -Er ?
    +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Json.Tests.ps1: line 53
53:             { ConvertTo-Json -InputObject $hash } | Should -Throw -ErrorId $expectedFullyQualifiedErrorId
```
ConvertTo-Json.Should handle null
```powershell
Value cannot be null. (Parameter 'type')
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\ConvertTo-Json.Tests.ps1: line 88
88:         $null | ConvertTo-Json -Compress | Should -Be 'null'

```